addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1073,6 +1073,7 @@ def _set_last_seen_message(self, last_message, allow_older=False, notify=True):
         })
         if notify:
             data = {
+                'reset_local_seen': allow_older,
```

Comment:
Name of prop is misleading: I thought at first it means that local seen is removed, but this actually means it forces syncing with `last_message_id`.

I suppose this is needed for multi-device/multi-tab purposes?

What about something like `force_local_seen_message: last_message.id`? Sure it feels a bit redundant in current payload that has `last_message_id`, but this would be less ambiguous thus easier to understand the meaning and actual action of this specific prop in isolation. The less guesses the better